source/help/getting-started/welcome-to-mattermost.rst

```diff
@@ -0,0 +1,130 @@
+Welcome to Mattermost!
+==========
+
+This article will go over the basics of Mattermost and a general overview of the appliation so that you can start using it right way.
+
+Ready? Let's get started!
+
+
+**What is Mattermost?**
+-----------------------------------
+
+Mattermost is a modern, digital workspace that will help you be more productive in your day to day work life.
+
+Some of the major benefits of using Mattermost are:
+
+- Direct 1:1 and group messaging with your colleages
+- Channels for topic-based, group-based, or meeting-based chat
+- Streamlined collaboration on projects
+- Reduced email clutter
+- Searching across messages and channels
+- Sharing files
+
+
+**The Basics**
+---------------------
+
+To understand how Mattermost works, below are some of the basic things you'll need to know. 
+
+This is a what a typical Mattermost workspace looks like. 
+
+.. image:: ../../images/ui3.png
+   :alt: ui
+
+
+These are the main features of Mattermost, we will be going over each one to understand how they work:
+
+- Teams
+- Channels
+- Messages
+- Search
+- Notifications
+
+
+
+**Teams**
+---------------------
+A team is a digital workspace where you and your teammates can collaborate on Mattermost. Depending on how Mattermost is set up in your organization, you can belong to one team or multiple teams.
+
+You have the ability to switch back and forth between teams on the Team sidebar, located on the upper left side of the app.
+
+.. image:: ../../images/ui_teams.png
+   :alt: ui_teams
+
+
+**Channels**
+---------------------
+Channels are used to organize conversations across different topics. They are located on the left-hand panel of Mattermost. 
+
+There are three types of channels: Public Channels, Private Channels, and Direct Messages.
+
+
+.. image:: ../../images/ui_channels2.png
+   :alt: ui_channels
+
+
+- **Public Channels:**
+
+Public Channels are open to everyone on a team. New team members are automatically added to two Public Channels when they sign up: Town Square and Off-Topic.
+
+Public Channels are identified with a **globe icon**.
+
+- **Private Channels:**
+
+Private Channels are for sensitive topics and are only visible to select team members. Any member of a Private Channel can add additional members. Channel members can choose to leave at any time, but only the channel owner or Team Admin can remove other members.
+
+Private Channels have a **lock icon**.
+
+- **Direct Messages (DM)**
+
+Direct Messages are for conversations between two people. 
+
+Group Messages are Direct Messages that have conversations among three or more people. Both are visible only to the people involved.
```

Comment:
Should we change the heading for this section to "Direct and Group Messages (DM / GM)" ?